templates/zerver/api/get-all-users.md

```diff
@@ -51,26 +51,7 @@ You may pass the `client_gravatar` query parameter as follows:
 
 #### Return values
 
-* `members`: A list of dictionaries, each containing the details for
-  one of the users in the organization.
-    * `email`: The email address of the user or bot.
-    * `is_bot`: A boolean specifying whether the user is a bot or not.
-    * `avatar_url`: URL to the user's gravatar. `None` if the `client_gravatar`
-      query parameter was set to `True`.
-    * `full_name`: Full name of the user or bot.
-    * `is_admin`: A boolean specifying whether the user is an admin or not.
-    * `bot_type`: `None` if the user isn't a bot. `1` for a `Generic` bot.
-      `2` for an `Incoming webhook` bot. `3` for an `Outgoing webhook` bot.
-      `4` for an `Embedded` bot.
-    * `user_id`: The ID of the user.
-    * `bot_owner_id`: If the user is a bot (i.e. `is_bot` is `True`),
-      `bot_owner` is the user ID of the bot's owner (usually, whoever
-      created the bot).  **Changes**: New in Zulip 2.2 (feature level
-      1).  In previous versions, there was a `bot_owner` field
-      containing the email address of the bot's owner.
-    * `is_active`: A boolean specifying whether the user is active or not.
-    * `is_guest`: A boolean specifying whether the user is a guest user or not.
-    * `timezone`: The time zone of the user.
+{generate_return_values_table|zulip.yaml|/users:get}
```

Comment:
In this one, the translation loses useful context in the `Changes` field; I think everything else is better in the OpenAPI file.  Though it'd be really nice to make the list of valid values for `bot_type` use a bulleted list while we're touching this.

Reply:
Done both.